src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService.cs

```diff
@@ -169,8 +169,8 @@ protected static T Cast<T>(object value)
                 !(destination is TDeclarationNode4))
             {
                 throw new ArgumentException(
-                    string.Format(WorkspacesResources.Destination_type_must_be_a_0_1_or_2_but_given_one_is_3,
-                        typeof(TDeclarationNode1).Name, typeof(TDeclarationNode2).Name, typeof(TDeclarationNode3).Name, typeof(TDeclarationNode4).Name),
+                    string.Format(WorkspacesResources.Destination_type_must_be_a_0_1_2_or_3_but_given_one_is_4,
```

Comment:
this should be undone right?

Reply:
I think this is an existing bug based on David's comment here: https://github.com/dotnet/roslyn/pull/39745#discussion_r345368803
It's not actually used in the extract local method code, but I thought I would just fix it in this PR.